### PR TITLE
Additional test for trestle add

### PR DESCRIPTION
**Pull request by aNebula**
## Types of changes

- \[x\] Bug fix (non-breaking change which fixes an issue)
- \[ \] New feature (non-breaking change which adds functionality)
- \[ \] Breaking change (fix or feature that would cause existing functionality to change)
- \[x\] My code follows the code style of this project.
- \[ \] My change requires a change to the documentation.
- \[ \] I have updated the documentation accordingly.
- \[x\] I have added tests to cover my changes.
- \[x\] All new and existing tests passed.
- \[ \] All commits are signed.

### CODE REVIEW

**Comment by aNebula** [comment]
hmmm... tests failing... doesn't look like the failing tests are to do `trestle add`. I'll have a look on Monday

**Comment by aNebula** [comment]
> hmmm... tests failing... doesn't look like the failing tests are to do `trestle add`. I'll have a look on Monday

Shouldn't have merged develop?

File: tests/trestle/core/commands/add_test.py [hunk 1]
```diff
@@ -111,13 +113,17 @@ def test_run_failure():
...
+        assert e.type == SystemExit
+        assert e.value.code == 2
 
     testargs = ['trestle', 'add', '-f', './catalog.json']
     with patch.object(sys, 'argv', testargs):
-        with pytest.raises(SystemExit):
+        with pytest.raises(SystemExit) as e:
             Trestle().run()
+        assert e.type == SystemExit
+        assert e.value.code == 2
```
Comment (butler54) [inline tests/trestle/core/commands/add_test.py:127]: Just want to check but looks like the argparse method will automatically throw a sys exit (inside of code we have not written)?

**Review by butler54 — APPROVED** [review]
LGTM - assuming my thoughts on the comment are correct.

**Pull request merged**